.gitattributes

```diff
@@ -39,4 +39,4 @@
 
 # Yarn
 # https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored
-/war/.yarn/plugins/** binary
+/.yarn/plugins/** binary
```

Comment:
Not related to this PR conceptually, but fixing a bug introduced in #9718 while I was here.